### DISCUSSION TITLE
SLAT state in Standalone installation

### DISCUSSION THREAD

**Discussion by abs0248** [Security-Onion-Solutions/securityonion#2305] [Unsupported Versions] [answered]
Hi.i installed standalone SO-2.3.10 and tried to tuning  IDS alert as the old way in SO-16.0.4 by using the `threshold.conf` . I read the new SO-2.X Docs , it  recommend to use SALT state instead of using the xyz.conf . i search for the path for any salt pillar  (both global.sis and minion.sis) and didn't find any in the file system. I want to know is  SALT in standalone installation still in charge of the configurations for tools or not because i didn't find even one *.sis in the  file system and even some paths that mentioned in the Docs does not exist.
Thanks

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=threshold#threshold:

> In order to apply the threshold to all nodes, place the pillar in /opt/so/saltstack/local/pillar/global.sls. If you want to apply the threshold to a single node, place the pillar in /opt/so/saltstack/local/pillar/minions/<MINION_ID>.sls

Please note that the file extension is `.sls` and NOT `.sis`.

**Reply by abs0248** [reply]
thanks for your response
